doc/tutorial/08-set.md

```diff
@@ -182,7 +182,7 @@ cleaner to have a field on the SetClient.
 
```

Comment:
Ah, this should have shown the full defrecord--e.g. `(defrecord SetClient [conn k])`. Like the text says, we're gonna add a key parameter.

Reply:
I see, updated with 'k' param